src/cargo/ops/cargo_new.rs

```diff
@@ -811,6 +811,8 @@ fn main() {
             b"\
 #[cfg(test)]
 mod tests {
+    // Uncomment this to use items from the containing module.
+    // use super::*;
```

Comment:
Small nitpick, this might be better as `//use super::*;`. The extra space could get missed